src/Symfony/Component/Form/FormFactory.php

```diff
@@ -61,36 +61,12 @@ public function createForProperty($class, $property, $data = null, array $option
      */
     public function createBuilder($type = 'Symfony\Component\Form\Extension\Core\Type\FormType', $data = null, array $options = array())
     {
-        $name = null;
-        $typeName = null;
-
         if ($type instanceof ResolvedFormTypeInterface) {
```

Comment:
Yeah, it must be removed. It was deprecated in 2.8: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Form/FormFactory.php#L114
So it must be removed from this method as well.